src/db/ObjectiveContributors/ObjectiveContributors.js

```diff
@@ -12,6 +12,17 @@ const create = async (itemRef, objectiveRef) => {
     throw new Error(`Cannot find objcetive with ID ${objectiveRef.id}`);
   }
 
+  const contributors = await db
+    .collection('objectiveContributors')
+    .where('objective', '==', objectiveRef)
+    .where('item', '==', itemRef)
+    .get()
+    .then((snapshot) => snapshot.docs.map((doc) => doc.ref));
+
+  if (contributors.length > 0) {
+    return undefined;
+  }
```

Comment:
Er dette for å hindre duplikater?

Burde i så fall kanskje kastet en `Error` her i stedet og håndtert det utenfor, men kan eventuelt endres på senere også.